sql.d/problems.sql

```diff
@@ -31,14 +31,13 @@ WHERE repo = %(repo)s;
 -- !!get_repository_problems(repo, limit=None) -> array of dicts
 --
 --------------------------------------------------------------------------------
-SELECT
-	repo,
-	name,
-	effname,
+SELECT 
+	packages.*,
 	maintainer,
 	problem
 FROM problems
-WHERE repo = %(repo)s
+INNER JOIN packages ON packages.id = problems.package_id
+WHERE packages.repo = %(repo)s
```

Comment:
Post-mortem note: this was not correct - you've had to use problems.repo here. Despite that the contents are the same, if you refer to `packages` for conditions or sorting, no the index scan on `problems` cannot be used and the database fallbacks to seq scan which leads to problems page answering much longer. Also you've missed the update to similar `get_maintainer_problems` query.